kong/llm/init.lua

```diff
@@ -184,12 +183,6 @@ _M.config_schema = {
                                       then_at_least_one_of = { "model.options.mistral_format" },
                                       then_err = "must set %s for mistral provider" }},
 
-    { conditional_at_least_one_of = { if_field = "model.provider",
```

Comment:
Not anymore - the model can be passed from the user. They wanted this so that they can use e.g. the Python OpenAI SDK and bring their own models.

If one is not specified either in plugin conf NOR request, we return 400.